lib/model/EetplanFactory.class.php

```diff
@@ -196,23 +199,23 @@ public function genereer($avond, $random = true) {
             }
 
             # deze sjaars heeft op deze avond een huis gevonden
-            $this->sah[$uid][$avond] = $huis_index;
+            $this->sah[$uid][$avond] = $huis_id;
             # en gaat alle sjaars die op deze avond in dit huis zitten dat melden
-            foreach ($this->ahs[$avond][$huis_index] as $sjaars) {
+            foreach ($this->ahs[$avond][$huis_id] as $sjaars) {
                 $this->bekenden[$uid][] = $sjaars; # alle sjaars in mijn seen
                 $this->bekenden[$sjaars][] = $uid; # ik in alle sjaars' seen
             }
-            $this->ahs[$avond][$huis_index][] = $uid;
+            $this->ahs[$avond][$huis_id][] = $uid;
             # de sjaars heeft het huis bezocht
-            $this->bezocht[$huis_index][] = $uid;
-            $this->bezocht_sh[$uid][$huis_index] = true;
-            $this->bezocht_ah[$avond][$huis_index][] = $uid;
+            $this->bezocht[$huis_id][] = $uid;
+            $this->bezocht_sh[$uid][$huis_id] = true;
+            $this->bezocht_ah[$avond][$huis_id][] = $uid;
 
             # Maak een entity voor deze sessie
             $nieuweetplan = new Eetplan();
             $nieuweetplan->avond = $avond;
             $nieuweetplan->uid = $uid;
-            $nieuweetplan->woonoord_id = $this->huizen[$huis_index]->id;
+            $nieuweetplan->woonoord_id = $huis_id;
```

Comment:
Dus je hebt nu woonoord id (=groep id) en eetplan woonoord id (=huis id) en wat is dan inex?